default/global_settings.txt

```diff
@@ -0,0 +1,6 @@
+
+PRODUCTION_QUEUE_FRONTLOAD_FACTOR
```

Comment:
Your plan to move away from using global_settings.txt certainly sounds fine; I just don't think we need to jump all the way to the endpoint in one leap (and I don't have time to get us all the way there in the immediate future).  I'll take a look at adding a shipyards list for this system name code you mention.